src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1353,6 +1355,34 @@ class ApplicationManagerImpl
    */
   bool IsLowVoltage();
 
+  /**
+   * @brief Allows to process postponed commands for application
+   * when itshmi level has been changed.
+   * @param app_id the application id for processing.
+   * @param from the old HMILevel.
+   * @param to the new HMILevel for the certain app.
+   */
+  void ProcessPostponedMessages(const uint32_t app_id);
+
+  /**
+   * @brief Allows to process navi applications after HMILevel has been changed.
+   * @param app_id the application id for processing.
+   * @param from the old HMILevel.
+   * @param to the new HMILevel for the certain app.
+   */
+  void ProcessNaviApp(const uint32_t app_id,
```

Comment:
@LitvinenkoIra please rename this method it is completely unclear why there is a separate function for processing Navi and no separate functions for others.